code/modules/reagents/chemistry/machinery/smoke_machine.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/smoke_machine
-	name = "Smoke Machine"
-	desc = "Seriously man?"
+	name = "smoke Machine"
+	desc = "A machine installed with a centrifuge inside of it. It generates smoke with whatever reagent you put into it."
```

Comment:
With a certifuge installed

Also if you wouldn't mind revising the last scentence a bit, it seems awkwardly worded.